src/getters/getCurrentChain.ts

```diff
@@ -0,0 +1,9 @@
+import { useChainStore } from '~/shared/stores/chain'
+
+export function getCurrentChain() {
+    return useChainStore.getState().selectedChain
```

Comment:
Haha, I didn't realize u can access the store from the outside that easily. Cheers!